project-anacapa/anacapa-github-linker
# Pc add repo test job

This PR adds a repo level test job as scaffolding for a future job that 

To test:
* Navigate to a course page as admin
* Click the "Repos" tab
* Select "All Repos"
* Select a specific repo to go to it's show page
* There should be a Job Launcher and Jobs Log panel
* Select to Run the test job
* You should see that there is a record of the test job being launched.

Note that the test job might be off by one hour in the elapsed time while running; this is known bug #270 
which will be fixed in a separate PR
